src/senselab/utils/data_structures/script_line.py

```diff
@@ -15,15 +16,15 @@ class ScriptLine(BaseModel):
 
     text: Optional[str] = None
     speaker: Optional[str] = None
-    start: Optional[float] = None
-    end: Optional[float] = None
+    start: Optional[float] = 0
```

Comment:
why this?

Reply:
conceptually, if there is no start or end, I do prefer to have None instead of 0.

Reply:
I agree. Undone. I think I did this because of a mypy issue or for testing something temporarily